pandas/tests/arrays/test_array.py

```diff
@@ -317,6 +317,26 @@ def test_bounds_check():
         pd.array([-1, 2, 3], dtype="UInt16")
 
 
+def test_ellipsis_index():
+    # GH42430
```

Comment:
this would go in tests.extension.base.getitem

Reply:
Moved to `tests/extension/base/getitem.py`